src/opensubtitles.html

```diff
@@ -0,0 +1,28 @@
+﻿<div id="openSubtitlesPage" data-role="page" class="page type-interior metadataConfigurationPage" data-require="scripts/opensubtitles,emby-input,emby-button,emby-linkbutton">
+    <div>
+        <div class="content-primary">
+            <form class="metadataSubtitlesForm">
+                <div style="height:0; overflow: hidden;"><input type="text" name="fakeusernameremembered" tabindex="-1" /><input type="password" name="fakepasswordremembered" tabindex="-1" /></div>
+
+                <div style="display: none;">
+                    <input type="text" id="PreventChromeAutocomplete" name="PreventChromeAutocomplete" autocomplete="address-level4" />
+                </div>
+                <br />
+                <div class="inputContainer">
+                    <input is="emby-input" type="text" id="txtOpenSubtitleUsername" autocomplete="off" label="${LabelOpenSubtitlesUsername}" />
+                    <div class="fieldDescription">
+                        <!-- TODO missing ${ButtonRegister} -->
```

Comment:
```suggestion
                        <!-- TODO missing ${ButtonRegister} localization string -->
```

Reply:
You could add it manually to the en-us strings right now so we at can avoid the key issue, translations will come later.